rita_common/src/debt_keeper/mod.rs

```diff
@@ -239,6 +240,13 @@ pub fn send_debt_update() -> Result<(), RitaCommonError> {
     // (mainly on exits) we batch tunnel change operations before sending them over
     let mut debts_message = Vec::new();
 
+    if potential_payment_issues_detected() {
```

Comment:
right thing to do, but this isn't the right place, this should be in make payment, we want to maintain the right balances in memory for when we need them (eg payments recover)